README.md

```diff
@@ -36,7 +36,7 @@ If anything feels off, or if you feel that some functionality is missing, please
 
 ### Linux
 
-`gh` is available via [Homebrew](#homebrew), and as downloadable binaries from the [releases page][].
+`gh` is available as downloadable binaries from the [releases page][].
```

Comment:
What does this correct, exactly? Can you elaborate? Is there something wrong with Homebrew on Linux?

Reply:
Something wrong? I wouldn't think so to begin with. I honestly, wanted to see? With actually trying a different format and or routes..? Maybe, just maybe? It might be able to open door's, that weren't there to begin with..? With change, doesn't mean that there's a problem..? More of, there's always more than 1 way to skin a cat..? Why not try? Something new is always worth a shot. And I could be a million percent wrong? But trying is better then "Not at all".